spring-cloud-data-rest/src/main/java/org/springframework/cloud/data/AdminApplication.java

```diff
@@ -14,7 +14,7 @@
  * limitations under the License.
  */
 
-package org.springframework.cloud.data.rest;
```

Comment:
We do not want to move the AdminApplication up. It belongs under 'rest' (the root of spring-cloud-data-rest). Why is this necessary?